packages/devtools_app/test/flutter/initializer_test.dart

```diff
@@ -2,18 +2,26 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+@TestOn('vm')
 import 'package:devtools_app/src/flutter/initializer.dart';
 import 'package:devtools_app/src/globals.dart';
 import 'package:devtools_app/src/service_manager.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+import '../support/mocks.dart';
+
 void main() {
   group('Initializer', () {
+    FakeServiceManager serviceManager;
```

Comment:
is this variable used? If not maybe revert this change and set it directly in the setGlobal call like it was before.

Reply:
No; done.